### DISCUSSION TITLE
I just want to enable middle mouse click, not click and drag. How do I do that?

### DISCUSSION THREAD

**Discussion by bendavis78** [noah-nuebling/mac-mouse-fix#426] [Questions] [answered]
Sometimes when I click the middle mouse and only drag a little bit, it pulls up mission control. I don't want that to happen. I just want to be able to use middle mouse click and nothing else. How do I disable the other gestures/bindings?

**Comment by noah-nuebling** [comment, marked as answer]
Hi @bendavis78, thanks for your question!

You can deleted a binding by selecting it and then clicking the "-" button at the bottom.

Hope that helps! Let me know if you have any more questions.
